src/apps/competitions/tasks.py

```diff
@@ -680,3 +680,13 @@ def create_competition_dump(competition_pk, keys_instead_of_files=True):
         logger.info(f"Finished creating competition dump: {temp_comp_dump.pk} for competition: {comp.pk}")
     except ObjectDoesNotExist:
         logger.info("Could not find competition with pk {} to create a competition dump".format(competition_pk))
+
+
+@app.task(queue='site-worker', soft_time_limit=60 * 5)
+def do_phase_migrations():
+    new_phases = Phase.objects.filter(auto_migrate_to_this_phase=True, start__lte=now(),
```

Comment:
should we do
```python
start__lte=now() + timedelta(minutes=30)
```
and start migrating a little before the next phase actually starts?

Reply:
I think this would prevent scoring for submissions made in the final 30 minutes of the phase, but not positive.

Reply:
Ah. Pretty sure you are right.